plugins/node/opentelemetry-instrumentation-graphql/src/instrumentation.ts

```diff
@@ -71,6 +74,17 @@ export class GraphQLInstrumentation extends InstrumentationBase {
       VERSION,
       Object.assign({}, DEFAULT_CONFIG, config)
     );
+    const ignoreMap = new Map();
+    config.ignoreFields?.map(item => ignoreMap.set(item, true));
+    this._setIgnoreMap(ignoreMap);
+  }
+
+  private _getIgnoreMap(): Map<string, boolean> {
```

Comment:
this seems to be over engineering , a private function that returns a private, I think you can get rid of this, unless there is very specific reason for that ?

Reply:
the reason is due to the wrapField and wrapFieldResolver functions. Since it can on the fly change the config, we need to make sure that those util functions get the updates. There are embedded functions so this will make sure we don't have to rerun that code.